react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -110,14 +123,19 @@ function BoxEditView() {
     // eslint-disable-next-line no-console
     console.log("boxFormValues", boxFormValues);
 
+    const tagIds = boxFormValues?.tags
+      ? boxFormValues?.tags?.map((tag) => parseInt(tag.value, 10))
+      : [];
+
     updateContentOfBoxMutation({
       variables: {
         boxLabelIdentifier: labelIdentifier,
         productId: parseInt(boxFormValues.productId, 10),
         numberOfItems: boxFormValues.numberOfItems,
-        comment: boxFormValues.comment,
+        comment: boxFormValues?.comment ? boxFormValues?.comment : "",
         sizeId: parseInt(boxFormValues.sizeId, 10),
         locationId: parseInt(boxFormValues.locationId, 10),
+        tagIds,
       },
     })
       .then((mutationResult) => {
```

Comment:
@vahidbazzaz I hope to see a PR where you also change the error handling away from the console using your global component.

Reply:
@HaGuesto It is actually outside the scope of this PR, but I will change it in the follow-up PRs.

Reply:
Just wanted to remind you :)